wiki/en/misc/1-index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Jamulus ‒ Play music online. With friends. For free."
+lang: "en"
+permalink: /
+layout: mainhomepage
+ldjAppCategory: "Music"
+ldjSlogan: "Play music online. With friends. For free."
+ldjCreator: "Volker Fischer and contributors"
+metadescription: "Jamulus is free and open source software that lets musicians perform music, rehearse or jam together, in real time over the Internet."
+mAltProgIcon: "Jamulus icon"
+mTSlogan: "Play music online. With friends. For free."
+mTGetStartedNow: "Get started now!"
+mTDownloadNow: 'Download now for'
+mTPlatformsAnd: 'and'
+mTOtherPlatforms: 'other platforms'
+---
+[//]: # Markdown sections should not be indented
+<div class="fx-row fx-row-center-xs" id="firstrow">
+  <div class="fx-col-100-xs">
+    <div itemprop="abstract">
+      <h2>What is Jamulus?</h2>
+<div markdown="1">
+Jamulus lets you play, rehearse, or jam with your friends, your band, or anyone you find online. Play together remotely in time with high quality, low-latency sound on a normal broadband connection. [Download it here!](wiki/Getting-Started)
+</div>
+    </div>
+  </div>
+</div>
+<div class="fx-row fx-row-center-xs" id="bannercontainer">
+  <div class="fx-col-100-xs">
+    <a href="wiki/Getting-Started">
+      <img alt="Jamulus Banner. Links to getting started page" src="{{ '/assets/img/jamulusbannersmall.png' | relative_url }}" id="jamulusbanner" loading="lazy" />
+    </a>
+  </div>
+</div>
+<div class="fx-row fx-row-center-xs">
+  <div class="fx-col-100-xs fx-col-50-l">
+     <h2>Jamulus worldwide</h2>
+<div markdown="1">
+All over the world Jamulus allows choirs to rehearse and rock bands to play. Jamulus brings folk and classical musicians together. It's being used for remote music lessons, in schools and universities, in private and in public — all in real time on the Internet, as if you were there in person.
+</div>
+    <h2>Help needed?</h2>
+      <p>
+<div markdown="1">
+Check out the [documentation](wiki/Getting-Started) and consider the [troubleshooting section](wiki/Client-Troubleshooting)!
```

Comment:
```suggestion
Check out [the documentation](wiki/Getting-Started) and consider the [troubleshooting section](wiki/Client-Troubleshooting)!
```

I try to favor larger hyperlinks rather than smaller ones because they're more noticeable and more clickable.

Reply:
Even go to "Check out the documentation" as the link?

Reply:
Maybe so.